package.json

```diff
@@ -50,8 +49,12 @@
     "uuid": "^8.3.2",
     "xml2js": "^0.4.23",
     "yargs": "^17.5.1",
+    "yn": "^3.1.1",
```

Comment:
is this a missing required package?

Reply:
just saw your comment.

Reply:
Yep, see https://github.com/coda/packs-sdk/pull/2204#discussion_r970138214